### PR TITLE
Slider

### PR DESCRIPTION
**Pull request by JulienDelemarle**

### CODE REVIEW

**Review by GeorgesFORTRY — APPROVED** [review]
ok

File: index.js [hunk 1]
```diff
@@ -47,6 +47,16 @@ app.get('/images', async (req, res) => {
   });
 });
 
+app.get('/slider', async (req, res) => {
```
Comment (LoicBrassart) [inline index.js]: Either you use `async` **and** `await`, or use classic callbacks

File: index.js [hunk 2]
```diff
@@ -47,6 +47,16 @@ app.get('/images', async (req, res) => {
...
+  connection.query('SELECT * FROM slider', (err, rows) => {
+  if (err) {
+      res.status(500).send('Error retrieving data from database !');
+    } else {
+      res.status(200).json(rows);
+    }
+  });
+});
+
 app.get('/members', async (req, res) => {
```
Comment (LoicBrassart) [inline index.js:60]: I see that I let it slide beforehand, please fix it everywhere

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
One asynchronous Evil

**Review by GeorgesFORTRY — APPROVED** [review]
ok

**Review by LoicBrassart — APPROVED** [review]
SnE!

**Pull request merged**